packages/core/src/validation.ts

```diff
@@ -21,7 +21,9 @@ import type {
   Notify
 } from './types'
 
-const chainId = Joi.string().pattern(/^0x[0-9a-fA-F]+$/)
+// const chainId = Joi.string().pattern(/^0x[0-9a-fA-F]+$/)
+const chainId = Joi.alternatives().
+                try(Joi.string().pattern(/^0x[0-9a-fA-F]+$/), Joi.number())
```

Comment:
```suggestion
const chainId = Joi.alternatives().
                try(Joi.string().pattern(/^0x[0-9a-fA-F]+$/), Joi.number().positive())
```

Reply:
Here we can enforce positivity with joi ☝🏾 🙌🏾

Reply:
@aaronbarnardsound @Adamj1232 Is there a reason for the duplicated validation code in `packages/common/src/validation.ts` (seen above)  and here? and is there a way to consolidate to make it DRYer?